build-custom-image/create-docker-secret.sh

```diff
@@ -52,7 +64,9 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: $secret
+  labels:
+    tensorstack.dev/resource: docker
 data:
-  config.json: $(echo $configJSON | base64 -w 0)
+  .dockerconfigjson: $(echo $configJSON | base64 -w 0)
 type: Opaque
```

Comment:
`kubernetes.io/dockerconfigjson`?

Reply:
it is of type `Opaque`, not `kubernetes.io/dockerconfigjson`